docs/developer-guide.md

```diff
@@ -48,6 +48,105 @@ For the development Docker image, use:
 make docker-dev
 ```
 
+## Customizing and Building the `cube-ai` UI Image
+
+This guide covers the steps required to customize the `cube-ai` UI, build a Docker image for it, and manually push it to GitHub Container Registry (GHCR).
+
+### Prerequisites
+
+- Access to the private `magistrala-ui-new` GitHub repository.
```

Comment:
Customers will not have this, so they must access to Cube UI dir, if we ever open it. But I think we will give open-source only services, without UI (UI will be closed-source, as is for Magistrala).

Reply:
So we need to thisnk about this, and when we decide we'll see if this is only private doc that should not be published, but keep only internally for the team.

Reply:
> So we need to thisnk about this, and when we decide we'll see if this is only private doc that should not be published, but keep only internally for the team.

I think in the meantime it can be kept as a private doc for internal project developments

Reply:
We will open-source this immediately, so please fix ASAP